### PR TITLE
V0.6 gxs limited sync2

### PR DESCRIPTION
**Pull request by csoler**

### CODE REVIEW

**Comment by csoler** [comment]
This branch is for testing. It shouldn't be merged yet!

File: libretroshare/src/gxs/rsgxsnetservice.cc [hunk 1]
```diff
@@ -4231,52 +4223,59 @@ void RsGxsNetService::handleRecvSyncMessage(RsNxsSyncMsgReqItem *item,bool item_
...
+                        for(std::list<RsNxsItem*>::const_iterator it(itemL.begin());it!=itemL.end();++it)
+                            delete *it ;
 
-				    itemL.clear() ;
-				    break ;
-			    }
-		    }
-		    else
-		    {
+                        itemL.clear() ;
```
Comment (sehraf) [inline libretroshare/src/gxs/rsgxsnetservice.cc:4234]: you don't accept patches that fix indention but you commit changes that break indention - do you ever check what you are commiting?

File: libretroshare/src/gxs/rsdataservice.cc [hunk 2]
```diff
@@ -890,22 +890,22 @@ int RsDataService::storeGroup(std::map<RsNxsGrp *, RsGxsGrpMetaData *> &grp)
         locked_clearGrpMetaCache(grpMetaPtr->mGroupId);
 
         if (!mDb->sqlInsert(GRP_TABLE_NAME, "", cv))
-	{
-		std::cerr << "RsDataService::storeGroup() sqlInsert Failed";
-		std::cerr << std::endl;
-		std::cerr << "\t For GroupId: " << grpMetaPtr->mGroupId.toStdString();
-		std::cerr << std::endl;
-	}
+    {
```
Comment (sehraf) [inline libretroshare/src/gxs/rsdataservice.cc:893]: again only (wrong) white space changes ...

File: libretroshare/src/gxs/rsgxsnetservice.cc (hunk 1 above)
Comment (csoler) [reply libretroshare/src/gxs/rsgxsnetservice.cc:4234]: The reason is simple: when I review a pull request, I cannot afford to waste time figuring out that modifications are just indentation modifications, which can easily hide small bugs and therefore still need my full attention and time.

Comment (sehraf) [reply libretroshare/src/gxs/rsgxsnetservice.cc:4234]: i'm curious what people are using for development - must be Windows editor or something similar.
Or they all have limited time resources - like you - that they cannot setup proper indention in their editor/IDE

**Pull request merged**